Utils/trigger_force_upload_pack_to_production.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+_branch=$1
+_circle_token=$2
```

Comment:
@yaakovi let's start documenting these scripts.
Please add documentation explaining how to run? What this script triggers and why do we need it, when we need to use it

Reply:
https://confluence.paloaltonetworks.com/display/DemistoContent/Marketplace+-+general+process+of+content+packs+upload